Kickstarter-iOS/Views/Controllers/BackerDashboardViewController.swift

```diff
@@ -0,0 +1,393 @@
+  @IBOutlet private weak var headerView: UIView!
+  @IBOutlet private weak var headerViewTopConstraint: NSLayoutConstraint!
+  @IBOutlet private weak var menuButtonsStackView: UIStackView!
+  @IBOutlet private weak var messagesButtonItem: UIBarButtonItem!
+  @IBOutlet private weak var savedContainerView: UIView!
+  @IBOutlet private weak var savedMenuButton: UIButton!
+  @IBOutlet private weak var selectedButtonIndicatorView: UIView!
+  @IBOutlet private weak var selectedButtonIndicatorLeadingConstraint: NSLayoutConstraint!
+  @IBOutlet private weak var selectedButtonIndicatorWidthConstraint: NSLayoutConstraint!
+  @IBOutlet private weak var settingsButtonItem: UIBarButtonItem!
+  @IBOutlet private weak var sortBar: ProfileSortBarView!
+  @IBOutlet private weak var topBackgroundView: UIView!
+
+  private weak var backedProjectsViewController: ProfileBackedProjectsViewController!
+  private weak var savedProjectsViewController: ProfileSavedProjectsViewController!
+
+  fileprivate let viewModel: BackerDashboardViewModelType = BackerDashboardViewModel()
+
+  private var isCollapsed: Bool = false
+
+  internal static func instantiate() -> BackerDashboardViewController {
+    return Storyboard.BackerDashboard.instantiate(BackerDashboardViewController.self)
+  }
+
+  internal override func viewDidLoad() {
+    super.viewDidLoad()
+
+    self.backedProjectsViewController = self.childViewControllers
+      .flatMap { $0 as? ProfileBackedProjectsViewController }.first
+
+    self.backedProjectsViewController.delegate = self
+
+    self.savedProjectsViewController = self.childViewControllers
+      .flatMap { $0 as? ProfileSavedProjectsViewController }.first
+
+    self.savedProjectsViewController.delegate = self
+
+    self.backedMenuButton.addTarget(self, action: #selector(backedButtonTapped), for: .touchUpInside)
+
+    self.savedMenuButton.addTarget(self, action: #selector(savedButtonTapped), for: .touchUpInside)
+
+    _ = self.messagesButtonItem
+      |> UIBarButtonItem.lens.targetAction .~ (self, #selector(messagesButtonTapped))
+
+    _ = self.settingsButtonItem
+      |> UIBarButtonItem.lens.targetAction .~ (self, #selector(settingsButtonTapped))
+
+    self.viewModel.inputs.viewDidLoad()
+  }
+
+  internal override func viewWillAppear(_ animated: Bool) {
+    super.viewWillAppear(animated)
+
+    self.viewModel.inputs.viewWillAppear(animated)
+  }
+
+  // swiftlint:disable:next function_body_length
+  internal override func bindViewModel() {
+    super.bindViewModel()
+
+    self.backedContainerView.rac.hidden = self.viewModel.outputs.backedProjectsAreHidden
+    self.savedContainerView.rac.hidden = self.viewModel.outputs.savedProjectsAreHidden
+    self.backerNameLabel.rac.text = self.viewModel.outputs.backerNameText
+    self.backerLocationLabel.rac.text = self.viewModel.outputs.backerLocationText
+    self.sortBar.rac.hidden = self.viewModel.outputs.sortBarIsHidden
+
+    self.viewModel.outputs.avatarURL
```

Comment:
We have that helper now, think this can just be `self.avatarImageView.rac.imageUrl = self.viewModel.outputs.avatarUrl`